torch_geometric/utils/convert.py

```diff
@@ -455,3 +455,90 @@ def from_cugraph(g: Any) -> Tuple[Tensor, Optional[Tensor]]:
         edge_weight = from_dlpack(df['weights'].to_dlpack())
 
     return edge_index, edge_weight
+
+
+def to_dgl(
+    data: Union["torch_geometric.data.Data", "torch_geometric.data.HeteroData"]
+) -> Any:
+    r"""Converts a :class:`torch_geometric.data.Data` or
+    :class:`torch_geometric.data.HeteroData` instance to a :obj:`dgl` graph
+    object.
+
+    Args:
+        data (torch_geometric.data.Data or torch_geometric.data.HeteroData):
+            The data object.
+    """
+    import dgl
+
+    from torch_geometric.data import Data, HeteroData
+
+    if isinstance(data, Data):
+        g = dgl.graph((data.edge_index[0], data.edge_index[1]))
```

Comment:
We should also take `adj_t` into consideration.

Reply:
@EdisonLeeeee Thank you for your suggestions. Can you elaborate a bit on this one? Not sure if I understand what you mean by considering `adj_t`.

Reply:
As there may be cases where data only contains the transposed sparse adjacency matrix `adj_t`, we need to take this into consideration as well. For example, `data` was transformed by `ToSparseTensor`.